Update the bundle analyzer activation steps (#5914)
wso2/identity-apps: docs/write-code/PERFORMANCE.md

:bulb: The analyzer will open in http://localhost:8889

> 1. Navigate to `console` application directory:

Reviewer: ```suggestion
1. Navigate to `console` application directory:

```

```shell
cd apps/console
```
2. Edit the `.env.local` file to enable the analyzer:
```
ENABLE_ANALYZER=true
```
3. Build Application:

> ```shell

Reviewer: ```suggestion

```shell
```

pnpm build
```

#### Analyze for My Account

:bulb: The analyzer will open in http://localhost:8888

> 1. Navigate to `myaccount` application directory:

Reviewer: ```suggestion
1. Navigate to `myaccount` application directory:

```

```shell
cd apps/myaccount
```

> 2. Edit the `.env.local` file to enable the analyzer:

Reviewer: ```suggestion
2. Edit the `.env.local` file to enable the analyzer:

```

```
ENABLE_ANALYZER=true
```

> 3. Build Application:

Reviewer: ```suggestion

3. Build Application:

```

```shell
pnpm build
```

> Following these steps will enable the analyzer, the resulting view will look something like the following.

Reviewer: ```suggestion
Following these steps will enable the analyzer and result in a view that would look like the following.
```
